godotproject/project.godot

```diff
@@ -17,8 +17,8 @@ _global_script_class_icons={
 
 config/name="Frog Goes To Space - Cooper Prototype"
 run/main_scene="res://Menus/MainMenu/MainMenu.tscn"
-config/use_custom_user_dir.Pi=true
-config/custom_user_dir_name.Pi="/boot/fgts-pi/gamedata"
+config/use_custom_user_dir=true
+config/custom_user_dir_name="fgts-data"
```

Comment:
I had tried to put the game data (save file) directory in /boot, but that doesn't work. Now, it will go into `~/.local/share/fgts-data` on the Pi, and `%appdata%/fgts-data` on Windows